recursion/src/test/java/FibonacciRecursiveImplementationTest.java

```diff
@@ -13,16 +13,16 @@
 
     private static Stream<Arguments> test_getFibonacciNumber() {
         return Stream.of(
-                of(0, 0),
-                of(1, 1),
+                of(1, 0),
```

Comment:
Are you sure about that?
https://en.wikipedia.org/wiki/Fibonacci_number

Reply:
ping

Reply:
fixed